.rat-excludes

```diff
@@ -15,15 +15,15 @@ TAGS
 RELEASE
 control
 docs
-docker.properties.template
-fairscheduler.xml.template
-spark-defaults.conf.template
+docker.properties.*
+fairscheduler.xml.*
+spark-defaults.conf.*
 log4j.properties
-log4j.properties.template
+log4j.properties.*
 metrics.properties
-metrics.properties.template
+metrics.properties.*
 slaves
-slaves.template
+slaves.*
```

Comment:
Yeah, per the JIRA, if the intent is to exclude `conf/spark-defaults.conf`, that's not actually in the distribution.